src/scroll_list/ScrollList.js

```diff
@@ -536,6 +543,16 @@ define(function(require) {
         },
 
         /**
+         * Gets the scale translator.
+         *
+         * @method ScrollList#getScaleTranslator
+         * @return {ScaleTranslator}
+         */
+        getPositionTranslator: function() {
+            return this._positionTranslator;
+        },
```

Comment:
Docstring is out of date. While your at it, 'tranlation' is misspelled on line 424 :)